### DISCUSSION TITLE
[Feature] Add photo captions in the slideshow view

### DISCUSSION THREAD

**Discussion by imler** [immich-app/immich#8710] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Immich currently supports photo descriptions (both imported from EXIF and added manually), but they aren’t visible at all during a slideshow.

I would like to see a black transparent bar at the bottom of the image with white-centered text of the image description.

There should be enough room to add more information as well, such as the date, copyright info, or author of a photo (#8713).

Probably not everyone is a fan of this, so this bar can be an optional setting.

Here is an example of how it looks in Photoprism:

![caption](https://github.com/immich-app/immich/assets/28065743/13525ce3-56fb-4bf3-8629-cac06c1b730f)

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by geist325** [comment]
I would also very much love something like this!

I also run Photoprism, but I'm also teaching my daughter about photography and using a picture site.
Immich is MUCH better, except for this one feature lol.

I can share an album to people for them to view photos/videos without having to log in, but there's no way for me to add context or flavor to the album items without captions!!!

**Comment by MkengineTA** [comment]
I would love this feature for mobile, as we share our family photos mainly through the mobile app.

**Comment by dad2mads** [comment]
I too would really like to see this feature created.  I purchased a Frame TV to display my wife's photo's and am looking for a slideshow tool that provides more control and flexibility as to what photos are displayed (filtered by event/trip/etc), timing (how long each photo is displayed), etc.  Also want to present captions similar to how described  above that would show info such as Trip , month/year , etc. pulled either from data table or from the photo's metadata.

**Comment by Tobiasaufgithub** [comment]
I´d love to see this feature as it gives more context to a photo.

**Comment by tonideleo** [comment]
Same, I would love to see an option for this. Sharing albums with friends and families where we can story-tell them the context while they scroll through them.

**Comment by Hannnes1** [comment]
This is something I miss from regular old physical photo albums. Looking through old photos and reading a short caption to understand the context for each photo. It's easy to forget where and why a photo was taken

**Comment by momowell** [comment]
Nobody is working on this useful feature?

**Reply by timonrieger** [reply]
#24627

**Comment by timonrieger** [comment]
#24627 has been merged and will be available in the next release!
